bodo/pandas/base.py

```diff
@@ -73,6 +75,24 @@ def from_pandas(df):
     if not isinstance(df, pd.DataFrame):
         raise TypeError("Input must be a pandas DataFrame")
 
+    if isinstance(df.columns, pd.MultiIndex):
+        raise BodoLibNotImplementedException(
+            "from_pandas(): Hierarchical column names are not supported in Bodo yet."
+        )
+    new_columns = []
+    for c in df.columns:
+        if isinstance(df[c], pd.DataFrame):
```

Comment:
Using `df.columns.has_duplicates` is simpler and more reliable. `columns` is an Index, which is sort of a set and should have this info internally I think.